sdk/eventhub/Azure.Messaging.EventHubs/src/EventSender.cs

```diff
@@ -205,5 +245,21 @@ protected EventSender()
         ///
         [EditorBrowsable(EditorBrowsableState.Never)]
         public override string ToString() => base.ToString();
+
+        /// <summary>
+        ///   Yeilds an <paramref name="eventData" /> as a single element
+        ///   enumerable, for use with the methods that operate on a batch
+        ///   of events.
+        /// </summary>
+        ///
+        /// <param name="eventData">The event to yield.</param>
+        ///
+        /// <returns>The requested event in the form of an enumerable.</returns>
+        ///
+        private IEnumerable<EventData> YieldEvent(EventData eventData)
```

Comment:
Nit - I think the name `AsEnumerable` is more common here, but I don't feel strongly as it's not public.

Reply:
I had spun on this a bit;  there was a bunch of opinions to both use `AsEnumerable` and not, though most of those were aimed at extension methods and confusion.   Given that this is private, can't hurt for clarity.   I'll tweak it.

Reply:
why not return an array? return new EventData[] {eventData};

Reply:
In retrospect,  I'm not really saving anything by the indirection.   I refactored to the array approach.